trustfall_stubgen/test_data/expected_outputs/use_reserved_rust_names_in_schema/adapter/schema.graphql

```diff
@@ -0,0 +1,56 @@
+schema {
+  query: RootSchemaQuery
+}
+directive @filter(
+  """
+  Name of the filter operation to perform.
+  """
+  op: String!
+  """
+  List of string operands for the operator.
+  """
+  value: [String!]
+) on FIELD | INLINE_FRAGMENT
+directive @tag(
+  """
+  Name to apply to the given property field.
+  """
+  name: String
+) on FIELD
+directive @output(
+  """
+  What to designate the output field generated from this property field.
+  """
+  name: String
+) on FIELD
+directive @optional on FIELD
+directive @recurse(
+  """
+  Recurse up to this many times on this edge. A depth of 1 produces the current
+  vertex and its immediate neighbors along the given edge.
+  """
+  depth: Int!
+) on FIELD
+directive @fold on FIELD
+directive @transform(
+  """
+  Name of the transformation operation to perform.
+  """
+  op: String!
+) on FIELD
+
+"""
+All the possible data types where querying can begin in this API.
+"""
+type RootSchemaQuery {
```

Comment:
Are there any cases I missed?

Reply:
probably good to add a few more test cases using keywords from here: https://doc.rust-lang.org/reference/keywords.html

Reply:
Still probably good to throw in a few more reserved keywords here.

Reply:
[fa63cb8](https://github.com/obi1kenobi/trustfall/pull/406/commits/fa63cb817ffdc9e6638b4a85ccea6e25588408a9)